### PR TITLE
Feature#28 create check in logic

### LINKED ISSUE #28: Create check in logic
# REQUIREMENTS
Create check in logic

### PR DESCRIPTION
**Pull request by guillermo-delprado**
# Description
- We create the attendance migration, model, controller and views to generate with the private number assigned to employees an a daily attendance, then we show this attendances in the attendances index route
image

### CODE REVIEW

**Review by luisgarciaes — APPROVED** [review]
Looks great

**Review by ever864 — APPROVED** [review]
Buen trabajo!

**Review by acorzo — APPROVED** [review]
Excelent

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+
+  <%= form_with url: "/input", method: :post do |form| %>
+    <div>
+      <label>Private Number</label>
+      <%= form.text_field(:private_number, placeholder: "Example: 123456") %>
+    </div>
+    <%= form.submit("Check In", class: "btn btn-success") %>
+  <% end %>
+</div>
+
```
Comment (OwenLobato) [inline app/views/dashboard/index.html.erb:19]: Try to leave only one blank

**Review by OwenLobato — APPROVED** [review]

**Pull request merged**